dev-server/documents/html/vitalsource-pdf.mustache

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>VitalSource book reader (PDF)</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+    </style>
+    <script type="module">
+      /**
+       * Custom element that holds the book content frame. The class name is not
+       * important but the `<mosaic-book>` tag name triggers the VitalSource integration.
+       */
+      class MosaicElement extends HTMLElement {
+        constructor() {
+          super();
+          this.attachShadow({ mode: 'open' });
+
+          // For now there is only one page.
+          const pageURLs = [
+            '/document/vitalsource-pdf-page'
+          ];
+
+          let pageIndex = 0;
+
+          const setPage = index => {
```

Comment:
The "book" consists of only a single page at present, so no navigation between pages is possible. The code is structured to make it easy to add new pages in future and controls to switch between them, similar to what the EPUB test document has.